src/libraries/System.Text.Json/src/System/Text/Json/Document/JsonElement.Parse.cs

```diff
@@ -0,0 +1,100 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System.Diagnostics;
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Text.Json
+{
+    public readonly partial struct JsonElement
+    {
+        /// <summary>
+        ///   Parses one JSON value (including objects or arrays) from the provided reader.
+        /// </summary>
+        /// <param name="reader">The reader to read.</param>
+        /// <returns>
+        ///   A JsonDocument representing the value (and nested values) read from the reader.
```

Comment:
Should this be JsonElement?

```suggestion
        ///   A JsonElement representing the value (and nested values) read from the reader.
```

Reply:
Yep. Thanks